WordPressKit/WordPressKit/MediaServiceRemoteXMLRPC.m

```diff
@@ -52,18 +52,19 @@ - (void)getMediaLibraryStartOffset:(NSUInteger)offset
                      NSAssert([responseObject isKindOfClass:[NSArray class]], @"Response should be an array.");
                      if (success) {
                          NSArray *pageMedia = [self remoteMediaFromXMLRPCArray:responseObject];
+                         NSMutableArray *resultMedia = [NSMutableArray arrayWithArray:media];
```

Comment:
We can still eliminate the mutable array entirely (and the `if` below), I think, if we just do:

`NSArray *resultMedia = [media arrayByAddingObjectsFromArray:pageMedia];`